release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -2954,7 +2966,7 @@ Updating these fields did not trigger an update to the control plane machines, a
 
 * Previously, {op-system-base-full} CoreOS templates that were shipped by the  Machine Config Operator (MCO) caused node scaling to fail on {rh-openstack-first}. This issue happened because of an issue with `systemd` and the presence of a legacy boot image from older versions of {product-title}. With this release, a patch fixes the issue with `systemd` and removes the legacy boot image, so that node scaling can continue as expected. (link:https://issues.redhat.com/browse/OCPBUGS-42577[*OCPBUGS-42577*])
 
-* Previously, a restart of a Cluster Version Operator (CVO) pod while the pod was initializing a syncing operation caused the guard of a blocked upgrade request to fail. This meant that the blocked upgrade request was unexpectedly accepted. With this release, a CVO pod postpones reconciliation of requests during initialization, so that the guard of a blocked upgrade requests persists after the CVO pod restarts. (link:https://issues.redhat.com/browse/OCPBUGS-42386[*OCPBUGS-42386*]) 
+* Previously, a restart of a Cluster Version Operator (CVO) pod while the pod was initializing a syncing operation caused the guard of a blocked upgrade request to fail. This meant that the blocked upgrade request was unexpectedly accepted. With this release, a CVO pod postpones reconciliation of requests during initialization, so that the guard of a blocked upgrade requests persists after the CVO pod restarts. (link:https://issues.redhat.com/browse/OCPBUGS-42386[*OCPBUGS-42386*])
```

Comment:
```suggestion
* Previously, if a Cluster Version Operator (CVO) pod restarted while the pod was initializing, a syncing operation caused the guard of a blocked upgrade request to fail. As a consequence, the blocked upgrade request was unexpectedly accepted. With this release, a CVO pod postpones the reconciliation of requests during initialization so that the guard of a blocked upgrade requests persists after the CVO pod restarts. (link:https://issues.redhat.com/browse/OCPBUGS-42386[*OCPBUGS-42386*])
```
I realize that this wording isn't the focus of this PR, but I noticed a couple of things that could be tweaked. I considered revising the first sentence to be more active (`Previously, if you restarted a CVO pod while the pod was initializing...`), but I wasn't sure if the pod could be restarted by some other entity than the user.

Reply:
"initializing a syncing operation" is important here this represent what the Operator was doing. How about:

> Previously, if a Cluster Version Operator (CVO) pod restarted while the pod was initializing a syncing operation, the guard for a blocked upgrade request failed.

Reply:
Sounds good!